packages/browser/__tests__/login/oidc/OidcLoginHandler.spec.ts

```diff
@@ -77,7 +77,7 @@ describe("OidcLoginHandler", () => {
       mockStorage({}),
       mockStorage({
         "solidClientAuthenticationUser:mySession": {
-          clientId: "https://some.app/registration",
+          clientId: "some client ID",
```

Comment:
Why change this from a URI to a non-URI, since isn't this test just checking that whatever was stored is just being passed through? If the intent is really to use a non-URI, I'd change the value from "some client ID" (which tells me nothing), to something like "value of client ID doesn't matter - just checking it gets passed through from storage to the handler", or something like that...
But if the client ID is generally meant to be a URI (in fact a WebID), then I'd change the value to be a URI, but still communicate the intent of the test, e.g., "https://clientAppWebID.com/value-stored-in-storage-gets-passed-to-handler".

Reply:
Yeah, sure - my suggestion was just to make a point really, i.e., I wouldn't have suffixed the example value with "...just checking it gets passed through from storage to the handle" either. I totally agree about not using the value as a full-blown comment, but I would use the value to succinctly show intent, so "value doesn't matter" is something I use a lot (i.e., it's very generic, but communicates intent (from the test's perspective)). The value "some client ID" works generally too (although I'd suggest "Client ID doesn't matter" is a just a tiny bit better again!), but just in this particular case it kinda jumped out at me 'cos I expected a client ID to be WebID - but in this case (i.e., a generic authn library), it can indeed be a WebID or any arbitrary string value, so it's fine.